db/seed/person.seed.ts

```diff
@@ -0,0 +1,21 @@
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient()
+
+export async function personSeed() {
+  console.log('Persons seed')
+
+  const insert = await prisma.person.createMany({
+    data: [
+      { firstName: 'Sofia', lastName: 'Vicheva', email: 'sofia@abv.bg', phone: '+359888910910', company: 'ABC', address: 'Stefan Karadja 5, Sofia', newsletter: true},
```

Comment:
Let's use emails with suffix `@example.com` so we don't accidentally hit somebody's mailbox :sweat_smile: